content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -30,8 +30,6 @@ To run the node conformance test, perform the following steps:
     Because the test framework starts a local control plane to test the kubelet,
     use `http://localhost:8080` as the URL of the API server.
     There are some other kubelet command line parameters you may want to use:
-  * `--pod-cidr`: If you are using `kubenet`, you should specify an arbitrary CIDR
-    to Kubelet, for example `--pod-cidr=10.180.0.0/24`.
```

Comment:
I think you _might_ need to specify a Pod CIDR with some CNI plugins - can anyone confirm either way?